syntax = "proto3";

service Showtime {
    rpc GetMovieByTime(Timestamp) returns (stream MovieID) {}
    rpc Showtimes(EmptyMessage) returns (stream Schedules) {}
}

message Timestamp {
    string time = 1;
}

message MovieID {
    string id = 1;
}

message Schedules {
    string time = 1;
    repeated string ids = 2;
}

message EmptyMessage {

}